resources/assets/coffee/react/_components/beatmap-basic-stats.coffee

```diff
@@ -55,6 +67,10 @@ export BeatmapBasicStats = ({beatmapset, beatmap}) ->
         title: osu.trans "beatmapsets.show.stats.#{stat}",
           if stat == 'total_length'
             hit_length: formatDuration(beatmap['hit_length'])
+          else if stat == 'bpm'
+            count: beatmapset.bpm
```

Comment:
Store raw/un-suffixed value, remove the conditional above, and format with `formatNumber`.